demisto_sdk/commands/init/initiator.py

```diff
@@ -778,20 +1126,29 @@ def change_template_name_script_py(
             f.write(py_file_data)
             f.truncate()
 
-    def rename(self, current_suffix: str):
+    def rename(
+        self,
+        current_suffix: str,
+        is_modeling_rules: bool = False,
+        is_parsing_rules: bool = False,
+    ):
         """Renames the python, description, test and image file in the path to fit the newly created integration/script
 
         Args:
             current_suffix (str): The yml file name (HelloWorld or HelloWorldScript)
         """
-        os.rename(
-            os.path.join(self.full_output_path, f"{current_suffix}.py"),
-            os.path.join(self.full_output_path, f"{self.dir_name}.py"),
-        )
-        os.rename(
-            os.path.join(self.full_output_path, f"{current_suffix}_test.py"),
-            os.path.join(self.full_output_path, f"{self.dir_name}_test.py"),
-        )
+        if os.path.exists(os.path.join(self.full_output_path, f"{current_suffix}.py")):
```

Comment:
please replace everything here with `Path` objects, it'll save ~30% of the code.
Hint: use `Path(...)with_suffix`